code/modules/cargo/blackmarket/blackmarket_items/tools.dm

```diff
@@ -80,3 +110,64 @@
 	price_max = 200
 	stock_max = 3
 	availability_prob = 50
+
+/datum/blackmarket_item/tool/thermal_eyepatch
+	name = "Thermal Eyepatch"
+	desc = "A thermal eyepatch, capable of tracking the heat signatures of living beings through solid objects."
+	item = /obj/item/clothing/glasses/thermal/eyepatch
+
+	price_min = 1000
+	price_max = 3000
+	stock = 1
+	availability_prob = 20
+
+/datum/blackmarket_item/tool/jumpboots
```

Comment:
not entirely sure if these should be player obtainable, it's definitely shitcode last time i checked

Reply:
Took a look at the file, and they dont seem too bad to me. But if you're certain if you want them gone, Ill give it the boot.